MAUI/ImageEditor/Shape Annotation.md

```diff
@@ -0,0 +1,160 @@
+---
+layout: post
+title: Shape Annotation in .NET MAUI ImageEditor control | Syncfusion
+description: Learn here all about the shape annotation feature of Syncfusion .NET MAUI ImageEditor (SfImageEditor) control and more.
+platform: MAUI
+control: SfImageEditor
+documentation: ug
+---
+
+# Shape Annotation in .NET MAUI ImageEditor (SfImageEditor)
+
+The image editor control allows you to adding various shapes with customizable settings.
```

Comment:
The image editor control allows you to add various shapes with customizable settings.

Reply:
Changed the content